extensions/integration-tests/src/main.ts

```diff
@@ -102,9 +102,22 @@ export function activate(context: vscode.ExtensionContext) {
 		}
 		while (true);
 	});
+
+	// This function is added as part of Onboarding Hackathon and will be used to add names
+	// by new memebers of the team as part of their first change in the repo
+	vscode.commands.registerCommand('test.showDevelopers', async () => {
```

Comment:
seems odd to me to have something like this around actual code. Why not have a test extension in samples or something that achieves the same purpose without diluting our actual source?

Reply:
What I suggested to Udeesha was : 

1. Have a HelloWorld.txt file or something that new people modify as part of the onboarding process. This will cover the scenario of getting them to do a full end-to-end checkin without modifying any of our actual code
2. Then have the step where they debug the command just use another command such as New Connection. This has the added benefit of letting them break into actual production code to see how it looks during normal usage

Reply:
In that case does it make more sense to have our README list active contributors (our team) and have new team members add their name to that list?

Reply:
Sure, whatever works. I believe this is going to be more than just people working on ADS though so didn't want to clutter up the readme with a huge list of names that we then have to maintain (if we care about it being accurate that is)

Reply:
I assumed it was onboarding for our team. A good story might be to have a test extension in our samples that people can checkin something then have them copy that to their extension directory (show them how extensions work), so they can checkin something while also seeing that same change in ADS.

Reply:
I actually also thought it's odd to do so. @udeeshagautam , you mentioned DSMain is also doing similar things, maybe we can sync up offline about this. it would be nice if we don't have to add the names. :)